.github/ISSUE_TEMPLATE/good_first_issue.md

```diff
@@ -0,0 +1,11 @@
+The purpose of the good first issue label is to highlight which issues are suitable for a new contributor without a deep understanding of the codebase.
```

Comment:
Missing the header:

```
---
name: Good fist issue
about: (Regular devs only): Suggest a new good first issue
title: ''
labels: goog first issue
assignees: ''

---

Reply:
Thanks! I will add this. GitHub seems to be in process of upgrading templates so I was uncertain whether it needed such a header. "You are using an old version of issue templates. Please update to the new issue template workflow"

Reply:
The header is the new workflow.

You can test this by making `michaelfolkson:20191101-first-issue` the default branch and enabling issues for your repo in the settings. Then, add the "good first issue" label from Bitcoin Core (needs to be capitalized in the same way)

Reply:
I've followed the above instructions but I can't seem to open a `good first issue` issue with the template that I've added as part of this PR. 

`good first issue` isn’t one of the options I see when creating a new issue (bug report, feature request, report a security vulnerability are the three buttons). “Edit templates” just directs me back to the ISSUE_TEMPLATE directory. Opening a blank issue and changing the label doesn’t update the template.

I’m assuming the issue creation experience needs to be customized with a config.yml file? https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository

Reply:
You'll have to change the default branch in https://github.com/michaelfolkson/bitcoin/settings/branches

Reply:
I did. I followed the above instructions.